### PR TITLE
New navigation

### PR DESCRIPTION
**Pull request by frigus02**
### Motivation

- Make all pages directly accessible (and discoverable) from the top navigation.
- Change URIs of pages to make them all top-level (e.g. `/about-us/people` is now just `/people`)

### Test plan

- Open all links from the top navigation and make sure the correct page is displayed.
- In addition for medium and large screens the currently active page should be underlined in the navigation. If the currently active page has child pages or is a child page, the corresponding sub navigation should stay visible.

Please note: The navigation on the Blog will be updated separately. For the Blog we just need to make sure the link is still correct.

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [x] Code review
  - [x] Design review
- [x] Manual testing
  - [x] Windows 7 IE 11
  - [x] Windows 10 Edge
  - [x] Mac OS El Capitan Safari
  - [x] Mac OS El Capitan Chrome
  - [x] Mac OS El Capitan Firefox
  - [x] iOS 9 Safari
  - [x] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [x] Navigate the site using the keyboard only
- [x] Tester approved

### Screenshots

New navigation on medium and large screens

New navigation on small screens

### CODE REVIEW

File: site/components/link/index.js [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
 import React, { PropTypes } from 'react';
+import { StateNavigator } from 'navigation';
 import { NavigationLink } from 'navigation-react';
 
-type LinkProps = { [id: string]: any };
-export default function Link(props: LinkProps) {
-  const { to, ...rest } = props;
+export default class Link extends React.Component {
+  static contextTypes = {
+    stateNavigator: () => {},
```
Comment (zoeabryant) [inline site/components/link/index.js]: How does this work?

File: site/components/link/test.js [hunk 2]
```diff
@@ -3,8 +3,18 @@ import React from 'react';
...
+    },
+  },
+};
+
 describe('components/link', () => {
   it('renders OK children', () => {
-    shallow(<Link to="foo">Hello</Link>);
+    shallow(<Link to="foo">Hello</Link>, {
+      context: mockContext,
+    });
```
Comment (zoeabryant) [inline site/components/link/test.js]: Could we have a test that covers the `hasActiveChild` functionality?

File: site/components/link/index.js (hunk 1 above)
Comment (frigus02) [reply site/components/link/index.js]: Our router ([navigation-react](https://github.com/grahammendick/navigation/tree/master/NavigationReact)) provides the `stateNavigator` in the react context. We can inject this into our component to get access to the currently active route.

Do we want a comment explaining this here?

File: site/components/link/test.js (hunk 2 above)
Comment (frigus02) [reply site/components/link/test.js]: Sure.

File: site/components/link/index.js (hunk 1 above)
Comment (zoeabryant) [reply site/components/link/index.js]: that may be nice - an empty function tends to shout 'unfinished' to me

File: site/components/link/test.js (hunk 2 above)
Comment (zoeabryant) [reply site/components/link/test.js]: You are also missing an expectation in 'renders OK children'

**Comment by frigus02** [comment]
Hi @declanslevin. This is ready for you to test now.

**Comment by declanslevin** [comment]
I noticed when testing accessibility that the highlight is being cropped when on the sub-navigation bar:
screen shot 2017-12-08 at 10 56 16
screen shot 2017-12-08 at 10 56 30
screen shot 2017-12-08 at 10 56 52

This doesn't happen with the main navigation bar - for consistency it would be good to fix this.

**Comment by declanslevin** [comment]
Testing with /52625e6/ the cropping on the sub navigation item accessibility highlights has been solved - however the highlight on the main navigation items is being masked by the sub navigation item below it.
screen shot 2017-12-08 at 13 47 36
screen shot 2017-12-08 at 13 47 14
This was actually happening previously but it wasn't so obvious because it entirely cropped a small amount from the bottom border (which you can see if you hover the mouse over either of the navigation items while it is highlighted).

Mac OS Chrome

**Pull request merged**